processing/src/test/java/org/apache/druid/segment/join/JoinConditionAnalysisTest.java

```diff
@@ -271,7 +283,7 @@ public void test_equals()
                           // These fields are tightly coupled with originalExpression
                           "equiConditions", "nonEquiConditions",
                           // These fields are calculated from nonEquiConditions
-                          "isAlwaysTrue", "isAlwaysFalse", "canHashJoin")
+                          "isAlwaysTrue", "isAlwaysFalse", "canHashJoin", "rightKeyColumns")
```

Comment:
The comment above is incorrect now. Actually, it was incorrect before too, since some of these fields are based on equiConditions as well.